openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -2498,13 +2350,13 @@ The technology described in this specification was made available from contribut
    * changes proof type descriptions to accommodate for the batch issuance changes
    * fix indentation of examples
    * changes proof type descriptions to accomodate for the batch issuance changes
-   * changed Batch Issuance endpoint to differentiate between issuance Credential Configuration, Credential Dataset and instances of Credentials with different key material
    * changed Credential Endpoint to enable requesting multiple instances of a particular Credential Configuration and Dataset with different cryptographic material
    * clarify optionality of scope and `authorization_details` for Authorization Request
-   * Clarify Batch Endpoint Encryption
+
    * Define Credential Format as a term
    * Define Credential Dataset as a term
    * Define Credential Configuration as a term
+   * Removes the Batch Credential Endpoint
```

Comment:
can you please add this to the top of the history?

Reply:
Will do.